nemo_skills/inference/prompt/openai/chat.yaml

```diff
@@ -0,0 +1,24 @@
+few_shot_examples:
+  template: "Example question:\n{question}\n\n{context}Example solution:\n{generation}\n\n\n\n\n\n"
```

Comment:
I changed the format here a bit, so the \n\n should go after {context} - can you please make it consistent with other prompt examples?

Reply:
I removed `\n\n` at all because the prompt will anyway be parsed and passed into openai api as a structured list